jail/conf.go

```diff
@@ -31,6 +31,7 @@ const (
   vnet.interface = "{{ join .VNetInterface ", " }}";
 {{- end }}
   persist;
+  vnet; // FIXME: this must not be hardcoded
```

Comment:
sic! need to figure out how to avoid hardcoding.

Reply:
In ocijail, I'm using annotations to let the container engine communicate what it needs to the OCI runtime. This should really be part of a FreeBSD extension to the OCI runtime spec but we don't have that yet.

Reply:
Lines 27-29 already have this handled as part of the existing extensions.  Can you use that?

Reply:
The original idea was to have something that works with nerdctl from the start, so I hardcoded it just for the sake of illustration. I guess we're going to standardize the extensions, so I fixed that, thanks!

Reply:
nerdctl can also use the extensions until they've been standardized.  That's what I've done with my ctr demos so far.